WORKSPACE

```diff
@@ -1,6 +1,9 @@
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
+# bazel_embedded has rules that support building for embedded targets.
+# We maintain a fork to build for RiscV32i
```

Comment:
Link to the fork?

Reply:
Down 4 lines: `remote = "<the fork>",` :)